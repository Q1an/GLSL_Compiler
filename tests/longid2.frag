textShorterThan1023CharsButLongerThan31
is
fine
t1234567890123456789012345abcdef

