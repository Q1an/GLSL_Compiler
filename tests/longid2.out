textShorterThan1023CharsButLongerThan31 line 1 cols 1-39 is T_Identifier (truncated to textShorterThan1023CharsButLong)
is           line 2 cols 1-2 is T_Identifier 
fine         line 3 cols 1-4 is T_Identifier 
t1234567890123456789012345abcdef line 4 cols 1-32 is T_Identifier (truncated to t1234567890123456789012345abcde)
